extensions/che-theia-hosted-plugin-manager-extension/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "extends": "../../configs/base.tsconfig.json",
     "compilerOptions": {
         "lib": [
-            "es6",
+            "ES2017",
```

Comment:
Since this is the default in `base.tsconfig.json`, I believe we can remove the "lib" option here. it would make future updates easier.

Reply:
done